styles.css

```
@@ -27,7 +27,7 @@ div {
 }
 
 img {
-    border-radius:5px;
+    border-radius:10px;
```

Increase border-radius 15px.
```suggestion
    border-radius:10px;
```